spec/definitions/DateRanges/custom.yaml

```diff
@@ -0,0 +1,18 @@
+description: Custom Date Range
+allOf:
+  -
+    $ref: "#/definitions/DateRange"
+  -
+    type: "object"
```

Comment:
We have [this](https://github.com/Rebilly/RebillyAPI/blob/master/spec/definitions/Scheduling/Methods/date-interval.yaml), could we re-use it? to keep API consistent?

Reply:
yes, looks good.  I suppose I will copy it to this repo